packages/utils/src/lib/report.ts

```diff
@@ -278,3 +279,22 @@ export function getPluginNameFromSlug(
     plugins.find(({ slug: pluginSlug }) => pluginSlug === slug)?.title || slug
   );
 }
+
+export function sortAuditIssues(a: Issue, b: Issue): number {
```

Comment:
Since it's not technically sorting, but comparing (i.e. intended to be passed as `compareFn` to `Array.sort`), then this would be a slightly better name, I think:

```suggestion
export function compareIssues(a: Issue, b: Issue): number {
```

Reply:
Renamed